docs/libcurl/opts/CURLINFO_CERTINFO.3

```diff
@@ -71,8 +71,9 @@ if(curl) {
 }
 .fi
 .SH AVAILABILITY
-This option is only working in libcurl built with OpenSSL, NSS, Schannel or
-GSKit support. Schannel support added in 7.50.0
+This option is only working in libcurl built with OpenSSL, NSS, Schannel, GSKit
+or Secure Transport support. Schannel support added in 7.50.0. Secure Transport
+support added in 7.78.0.
```

Comment:
This won't make it into 7.78.0, so it should sat 7.79.0. That being said, I think we need a more structured way to document the version availability than free text like this - that's not the responsibility of this patch but it's worth to start thinking about.